### PR TITLE
agentclusterinstall: adjust delete() returns

### PR DESCRIPTION
**Pull request by sebrandon1**
Broken out from: #379 

Making these into smaller PRs for easy review-ability.

### CODE REVIEW

File: pkg/assisted/agentclusterinstall.go [hunk 1]
```diff
@@ -638,12 +638,14 @@ func (builder *AgentClusterInstallBuilder) Delete() error {
 		builder.Definition.Name, builder.Definition.Namespace)
 
 	if !builder.Exists() {
-		return fmt.Errorf("agentclusterinstall cannot be deleted because it does not exist")
```
Comment (klaskosk) [inline pkg/assisted/agentclusterinstall.go:641]: Could you add a glog statement here please? I remember this came up in an earlier PR and I think the conclusion was to add them going forward

Comment (sebrandon1) [reply pkg/assisted/agentclusterinstall.go:641]: Done!  And I adjusted all of the PRs accordingly.  I forgot about adding the glog.

Comment (klaskosk) [reply pkg/assisted/agentclusterinstall.go:641]: Thanks!

**Review by klaskosk — APPROVED** [review]
/lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**